examples/secured-service-with-basic-auth/secured_service_with_basic_auth.server.out

```diff
@@ -2,8 +2,8 @@
 # into it, and use the `ballerina run` command to start the service.
 # Make sure the `sample-users.toml` file is populated correctly with user
 # information and located in the same directory as the
-# `secured_service_with_basic_auth.bal` file. Or else, the `sample-users.toml`
-# file can be provided as a runtime parameter, with the `--config` option.
+# `secured_service_with_basic_auth.bal` file. Else, the `sample-users.toml`
+# file can be provided as a runtime parameter using the `--config` option.
```

Comment:
"Make sure the `sample-users.toml` file is populated correctly with user information and located in the same directory as the `secured_service_with_basic_auth.bal` file. Or else, the `sample-users.toml` file can be provided as a runtime parameter, with the `--config` option." -> "Ensure that the `sample-users.toml` file is populated correctly with user information and provide it as the configuration file using the `--config` flag." 

Or something like that. Basically, the whole thing about being in the same directory is not applicable here.

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-examples/pull/766/commits/c451b9f9489bae3a82eb5220b18b5ff8a862ef3c